src/main/java/zpi/attraction/entity/Attraction.java

```diff
@@ -0,0 +1,61 @@
+package zpi.attraction.entity;
+
+import jakarta.persistence.*;
+import lombok.Getter;
+import lombok.ToString;
+
+@Getter
+@ToString
+@Entity
+@Table(name = "attractions")
+public class Attraction {
+    @Id
+    @GeneratedValue(strategy = GenerationType.IDENTITY)
+    @Column(name = "id")
+    private Integer id;
+
+    @Column(name = "district_id")
```

Comment:
Bez sensu trzymać districtId i districtName. Trzeba to wydzielić do innej tabeli